src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -78,7 +78,7 @@ $SENTRY_CLI debug-files upload "$INCLUDE_SOURCES_FLAG" "$DWARF_DSYM_FOLDER_PATH"
 ```
 
 For bitcode enabled builds via iTunes Connect, additional steps are required.
-Follow the instructions in [Sentry's bitcode documentation](/clients/cocoa/dsym/#dsym-with-bitcode) to set up uploads of symbols for all build variants.
+Follow the instructions in [Sentry's bitcode documentation](/platforms/apple/dsym/) to set up uploads of symbols for all build variants.
```

Comment:
@krystofwoldrich maybe we should even get rid of this section (or rather link to https://docs.sentry.io/platforms/apple/dsym/#dsym-with-bitcode) since bitcode is deprecated

Reply:
I also think we can remove it.
https://github.com/getsentry/sentry-docs/pull/7142#discussion_r1228357628